### ISSUE TITLE
Chapter 7 (classification II) further improvements

### ISSUE THREAD

**Issue by ttimbers** [UBC-DSCI/introduction-to-datascience#93]
### Tiffany's suggested improvements from reading it in the 2021 spring:
- In the overview we refer to the things we are tuning in this chapter as model parameters, however, I think technically we are tuning model hyperparameters...
   - Added new issue: #229  
- In step 4 of Evaluating accuracy, we should split the code block up below into generating predictions, and then looking at them, and then using `bind_cols` to to add the labels and then looking again how the data frame changed. I think it will be hard for newcomers to digest to two steps here since they have not looked at `cancer_test` for a few paragraphs now. 
    - *ML: I think this has been changed since this comment was made* 
- ~In step 5 of Evaluating accuracy, we should explain that kap is short for Cohen's kappa, another metric for classification that is beyond the scope of this course, but is less sensitive to imbalanced data than accuracy is.~
- ~Add whitespace here: "This says that the classifier labelled 43+82 = 125 observations correctly, " > "This says that the classifier labelled 43 + 82 = 125 observations correctly, "~
- In the "Tuning the classifier" section we should actually do this, use a validation set to pick K. Then discuss how we then retune the model with that K on the whole training set, then use that model to get the test error. The pose the question - what if we split the training and validation set differently? Would we have picked the same k? That leads into CV nicely.
   - *ML: I am not sure about this, after reading some of the reviewers' comments and/or hearing from students the process in the chapter is a little confusing, I am not sure doing this extra step adds much in terms of learning/demonstrating something interesting/contrasting and may lead to more confusion. Happy to discuss though!*  
- ~I find the first note in the "Cross-validation" section a bit confusing.~
- when you set `grid = 10` in `tune_grid`, why does it skip 4, 8 and 11?
   - *ML: I think this has changed since this comment was made* 
- ~The splitting data section needs to be fleshed out. It is just two definitions out of context.~

**Comment by ttimbers** [comment]
- why are we using a loop in section 7.4.1 - we do not teach this syntax - we should do this in a way they will understand,

**Comment by leem44** [comment]
All done or made into new issues
